ccmlib/cmds/node_cmds.py

```diff
@@ -370,15 +370,19 @@ def validate(self, parser, options, args):
         Cmd.validate(self, parser, options, args, node_name=True, load_cluster=True)
 
     def run(self):
-        self.node.dsetool(self.dsetool_cmd)
+        stdout, stderr, rc = self.node.dsetool(self.dsetool_cmd)
+        print_(stderr)
+        print_(stdout)
 
 
 class NodeDsetoolCmd(_DseToolCmd):
     usage = "usage: ccm node_name dsetool [options]"
     descr_text = "Run dsetool (connecting to node name)"
 
     def run(self):
-        self.node.dsetool(" ".join(self.args[1:]))
+        stdout, stderr, rc = self.node.dsetool(" ".join(self.args[1:]))
```

Comment:
The `nodetool` equivalent for this looks like ```stdout, stderr, rc = self.node.nodetool(self.nodetool_cmd + " " + " ".join((self.args[1:])))
```
So, it prepends the `*_cmd` attribute, where the `dsetool` version doesn't have that. I see that it's been that way, but it threw me off.